### PR TITLE
create Mongo-Db-Cheatsheet and updated readme 

### PR DESCRIPTION
**Pull request by ritwikdalmia**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ A Collection of programming [CheatSheets](./CheatSheets/) for developers to quic
 25. [pandas-cheatsheet.md](./CheatSheets/pandas-cheatsheet.md)
 26. [numpy-cheatsheet.md](./CheatSheets/Numpy-cheatsheet.md)
 27. [angular-cheatsheet.md](./CheatSheets/angular-cheatsheet.md)
+28. [Mongo-DB-cheatsheet.md](./CheatSheets/Mongo-DB-cheatsheet)
```
Comment (crescentpartha) [inline readme.md:61]: This link `doesn't work`. Please, use the `appropriate link`. 
```suggestion
28. [mongodb-cheatsheet.md](./CheatSheets/mongodb-cheatsheet.md)
```
Before this modification, please, rename the file `Mongo-DB-cheatsheet.md` to `mongodb-cheatsheet.md`.

File: CheatSheets/Mongo-DB-cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
+---
+title: PHP CheatSheet
+description: The most commonly used php concepts are given here.
+created: 2022-10-25
+---
```
Comment (crescentpartha) [inline CheatSheets/Mongo-DB-cheatsheet.md:5]:
```suggestion
---
title: MongoDB CheatSheet
description: The most commonly used MongoDB commands are given here.
created: 2022-10-26
---
```

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Everything looks to me good! Please, need some required changes.

**Comment by ritwikdalmia** [comment]
Check now

**Comment by ritwikdalmia** [comment]
please merge this PR

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Your `mongodb-cheatsheet.md` file **_doesn't exist_** in your own repository. That's why your **_link doesn't work_** and make a `conflict`. I close this PR and make a `new issue` for you. Please, create a `new pull request` under that issue.

**Pull request closed without merging**